python/lsst/ts/fbs/utils/maintel/basis_functions.py

```diff
@@ -207,6 +208,71 @@ def get_basis_functions_ddf_survey(
     ]
 
 
+def get_basis_functions_field_survey(
+    nside: int,
+    wind_speed_maximum: float = 10.0,
+) -> list[basis_functions.BaseBasisFunction]:
+    """Get the basis functions for a field survey.
+
+    Parameters
+    ----------
+    nside : `int`
+        The nside value for the healpix grid.
+    wind_speed_maximum : `float`
+        Maximum wind speed tolerated for the observations of the survey,
+        in m/s.
+
+    Returns
+    -------
+    `list` of `basis_functions.BaseBasisFunction`
+    """
+    sun_alt_limit = -12.0
+    moon_distance = 30.0
```

Comment:
Kind of picky, but I found later on that I liked having these parameters exposed at the ts_config_ocs level for AuxTel....I think moon contamination is a bigger concern for ComCam so I don't see us going closer than 30 deg, but it might be nice to have it adjustable.